subsys/net/lib/nrf_cloud/src/nrf_cloud_transport.c

```diff
@@ -543,6 +543,28 @@ static int nct_settings_init(void)
 }
 
 #if IS_ENABLED(CONFIG_NRF_CLOUD_FOTA)
+/* TODO: remove after testing */
+static void nrf_cloud_fota_ble_cb_handler(const struct nrf_cloud_fota_ble_job *
+					  const ble_job)
+{
+	LOG_INF("FOTA for BLE ID: %X:%X:%X:%X:%X:%X",
+		ble_job->ble_id.val[5],
+		ble_job->ble_id.val[4],
+		ble_job->ble_id.val[3],
+		ble_job->ble_id.val[2],
+		ble_job->ble_id.val[1],
+		ble_job->ble_id.val[0]);
+	LOG_INF("Job ID: %s, type: 0x%X, size: %d",
+		log_strdup(ble_job->info.id),
+		ble_job->info.type,
+		ble_job->info.file_size);
+	LOG_INF("File path: %s/%s",
+		log_strdup(ble_job->info.host),
+		log_strdup(ble_job->info.path));
+	nrf_cloud_fota_ble_job_update(ble_job, NRF_FOTA_SUCCEEDED);
```

Comment:
I assume I need to call `nrf_cloud_fota_ble_job_update()` in my real handler, correct?

Reply:
you would call `nrf_cloud_fota_ble_job_update` whenever you wanted to update the BLE job... it probably wouldn't be in the handler, unless you know the device is ready to start FOTA.